### PR TITLE
Added three metrics

### PR DESCRIPTION
**Pull request by joostverhoog**
mx.odata.consume.created, ~.updated and ~.deleted were introduced in Studio Pro 9.13

### CODE REVIEW

**Comment by Adam-Dupaski** [comment]
@MarkvanMents @johannahemminger between Runtime and Integration components, please see who should review

File: content/en/docs/refguide/runtime/metrics.md [hunk 1]
```diff
@@ -405,8 +405,11 @@ The Runtime Server produces the following metrics out-of-the-box:
...
-| `mx.odata.publish.objects` | counter | `entity` | The total number of OData objects that were served for a particular type of object (`entity`). |
+| `mx.odata.retrieve` | counter | `entity` | The total number of objects of a certain entity type (`entity`) that were retrieved from an [OData service](/refguide/consumed-odata-service). (introduced in Studio Pro 9.12) |
+| `mx.odata.consume.created` | counter | `entity` | The total number of objects of a certain entity type (`entity`) that were created using the [Send External Object activity](/refguide/send-external-object). (introduced in Studio Pro 9.13) |
+| `mx.odata.consume.updated` | counter | `entity` | The total number of objects of a certain entity type (`entity`) that were updated using the [Send External Object activity](/refguide/send-external-object). (introduced in Studio Pro 9.13) |
+| `mx.odata.consume.deleted` | counter | `entity` | The total number of objects of a certain entity type (`entity`) that were created using the [Delete External Object activity](/refguide/delete-external-object). (introduced in Studio Pro 9.13) |
```
Comment (MarkvanMents) [inline content/en/docs/refguide/runtime/metrics.md:411]: @joostverhoog 
These three lines seem to be contradicted by #4551 when you say the same metrics are introduced in 9.**14**.

Do they need to go from this PR, or can #4551 be closed?

Comment (joostverhoog) [reply content/en/docs/refguide/runtime/metrics.md:411]: This pull request is about *consume* metrics, https://github.com/mendix/docs/pull/4551 is about *publish* metrics, those are two separate metrics

Comment (MarkvanMents) [reply content/en/docs/refguide/runtime/metrics.md:411]: Oh yes, they looked too similar and I missed that.
Thanks for clarifying.

**Review by MarkvanMents — APPROVED** [review]
Looks good to go - can it be published immediately?

**Comment by MarkvanMents** [comment]
Since this is for 9.13, I'll publish immediately.

**Comment by MarkvanMents** [comment]
Since this is for 9.13, I'll publish immediately.

**Pull request merged**